src/redux/actions.ts

```diff
@@ -1251,3 +1253,19 @@ export const addSortForNewSpreadsheet = (
         value,
     },
 });
+
+export const STATEESTIMATION_RESULT_FILTER = 'STATEESTIMATION_RESULT_FILTER';
+export type StateEstimationResultFilterAction = Readonly<Action<typeof STATEESTIMATION_RESULT_FILTER>> & {
+    filterTab: keyof AppState[typeof STATEESTIMATION_RESULT_STORE_FIELD];
+    [STATEESTIMATION_RESULT_STORE_FIELD]: MutableUnknownArray;
+};
+export function setStateEstimationResultFilter(
+    filterTab: keyof AppState[typeof STATEESTIMATION_RESULT_STORE_FIELD],
+    stateEstimationResultFilter: MutableUnknownArray
+): StateEstimationResultFilterAction {
+    return {
+        type: STATEESTIMATION_RESULT_FILTER,
+        filterTab: filterTab,
+        [STATEESTIMATION_RESULT_STORE_FIELD]: stateEstimationResultFilter,
+    };
+}
```

Comment:
if it does not work and is not requested by PO, maybe we should rollback everything dealing with sort/filter in redux ?

Reply:
Sorting and filtering removed